test/distributed/org/apache/cassandra/distributed/test/UpdateSystemAuthAfterDCExpansionTest.java

```diff
@@ -113,7 +115,7 @@ static public void beforeClass() throws Throwable
     {
         // reduce the time from 10s to prevent "Cannot process role related query as the role manager isn't yet setup."
         // exception from CassandraRoleManager
-        System.setProperty("cassandra.superuser_setup_delay_ms", "0");
+        CassandraRelevantProperties.SUPERUSER_SETUP_DELAY_MS.setLong(0);
```

Comment:
I've imported this enum as static.

Initially, the `AuthKeyspace#SUPERUSER_SETUP_DELAY` constant has the `long` type [1], so it will look a bit strange if we change here method call from `setLong` to `setInt`, however it will work without errors. I also think this constant may be related to the public API and we are not able to change it.

[1] https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/auth/AuthKeyspace.java#L58

Reply:
ok keep it like it is